alacritty/src/main.rs

```diff
@@ -130,10 +130,16 @@ fn run(
 ) -> Result<(), Box<dyn Error>> {
     info!("Welcome to Alacritty");
 
-    info!("Configuration files loaded from:");
-    for path in &config.ui_config.config_paths {
-        info!("  \"{}\"", path.display());
-    }
+    info!(
+        "{}",
+        &config.ui_config.config_paths.iter().fold(
+            String::from("Configuration files loaded from:"),
+            |mut msg, path| {
+                msg.push_str(&format!("\n  {:?}", path.display()));
+                msg
+            },
+        )
```

Comment:
This is getting a bit messy. It should at least be extracted into a separate statement.